src/browser.ts

```diff
@@ -101,6 +132,20 @@ export class GrpcClient {
     );
   }
 
+  /**
+   * Browser version of createStub
+   * Creates a gRPC-fallback stub and auth
+   * @param {function} CreateStub - The constructor function of the stub.
+   * @param {Object} service - [TODO]
+   * @param {Object} options - The optional arguments to customize
```

Comment:
it's `opts`, not `options`. Should either change the code, or the JsDoc.

Reply:
Let's just say `Connection options, as described below.`, without mentioning gRPC.